components/util/opts.rs

```diff
@@ -223,6 +226,9 @@ pub struct DebugOptions {
     /// Profile which events script tasks spend their time on.
     pub profile_script_events: bool,
 
+    /// Enable all heartbeats for profiling.
+    pub profile_heartbeats: bool,
```

Comment:
Done. Also added missing line for `profile-script-events`.
Moved `convert-mouse-to-touch` to be in the same order as it is above.